CMakeLists.txt

```diff
@@ -17,10 +17,12 @@ find_package(catkin REQUIRED COMPONENTS
   franka_control
   franka_gripper
   imu_filter_madgwick
+  message_generation
```

Comment:
What is the message_generation package and what does it do? Did we create it? If so, I think we should rename it to better represent what it does.